### PR TITLE
[SDKs] Node.js improvement - Part 1

### PR DESCRIPTION
**Pull request by eozores-telefonica**
### :tickets: JIRA ticket
[LATCH-2217](https://jira.tid.es/browse/LATCH-2217)

### :goal_net: What's the goal?
Update access url
Update copyright
Add new endpoints

### :construction: How do we do it?
_Provide a description of the implementation. A list of steps would be ideal._
* Update copyright
* Update urls to latch.tu.com
* Update latch with totps and control-status
* Update project config, test and added .gitignore

### :blue_book: Documentation changes?
- [x] No docs to update nor create

### :test_tube: How can I test this?
Download project and execute:
* ```npm test```
* ```node examples\example.js```

### Last words
- It's a minor upgrade adding new funcionality, should be backward compatible

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -182,6 +182,33 @@ var latch = {
         }
         url += "/i/" + instanceId;
         _http("DELETE", url, '', '', '', next);
+    },
+
+    createTopt: function(userId, commonName, next) {
```
Comment (Bocanegra) [inline index.js:187]: Topt -> TOTP

File: index.js [hunk 2]
```diff
@@ -182,6 +182,33 @@ var latch = {
...
+        let params = { userId, commonName };
+        let url = config.API_TOTP_URL;
+        _http("POST", url, params, '', '', next);
+    },
+
+    getTopt: function(totpId, next) {
```
Comment (Bocanegra) [inline index.js:193]: The same, Topt -> TOTP

File: index.js [hunk 3]
```diff
@@ -182,6 +182,33 @@ var latch = {
...
+        let url = `${config.API_TOTP_URL}/${totpId}`
+        _http("GET", url, '', '', '', next);
+    },
+
+    validateTopt: function(totpId, code, next) {
```
Comment (Bocanegra) [inline index.js:198]: The same

File: test/latchSpec.mjs [hunk 4]
```diff
@@ -65,10 +65,10 @@ describe("latch", function(){
...
 
             latch.init(args);
-            var response = latch.status('1234', function(err, result) {
+            var response = latch.status('1234', '', '', function(err, result) {
                 expect(result).to.have.a.property('error').that.is.an('object');
-                expect(result).to.have.deep.property('error.code', 102);
-                expect(result).to.have.deep.property('error.message', 'Invalid application signature');
+                expect(result.error).to.have.a.property('code', 102);
+                expect(result.error).to.have.a.property('message', 'Invalid application signature');
                 done();
```
Comment (Bocanegra) [inline test/latchSpec.mjs:72]: I would like more tests

**Review by alfred82santa — APPROVED** [review]

**Pull request merged**